membership-common/src/test/scala/com/gu/memsub/subsv2/services/SubscriptionServiceTest.scala

```diff
@@ -31,27 +31,26 @@ import utils.TestLogPrefix.testLogPrefix
   */
 class SubscriptionServiceTest extends Specification {
 
-  // this is the UAT prpId of friend, which we need for the catalog
-  val prpId = ProductRatePlanId("2c92c0f94cc6ea05014cdb4b1d1f037d")
+  val digipackAnnualPrpId = ProductRatePlanId("2c92c0f94bbffaaa014bc6a4212e205b")
   val partnerPrpId = ProductRatePlanId("2c92c0f84c510081014c569327003593")
-  val supporterPrpId = ProductRatePlanId("2c92c0f84bbfeca5014bc0c5a793241d")
+  val supporterPlusPrpId = ProductRatePlanId("8ad08cbd8586721c01858804e3275376")
```

Comment:
I decided to update these tests to use S+ and digi sub, rather than friend and supporter.